go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/wader/gormstore
+module github.com/wepala/gormstore
```

Comment:
Shouldn't be included i guess? and same for v2/go.mod

Reply:
I'm sorry. I was trying to get the repo to work for my company's project. I reverted the change of the name of the module. I changed the imports version number since it was causing some issues: gorm.io and drivers for gorm like mysql, postgres and sqlite3. It would be much appreciated if you could make those changes to your repo.